tests/phpunit/tests/rest-api/rest-users-controller.php

```diff
@@ -856,10 +856,83 @@ public function test_get_items_invalid_roles() {
 		$request->set_param( 'roles', 'steakisgood' );
 		$response = rest_get_server()->dispatch( $request );
 		$data     = $response->get_data();
-		$this->assertCount( 0, $data );
-		$this->assertSame( array(), $data );
+		$this->assertIsArray( $data );
+		$this->assertEmpty( $data );
+	}
+
+	public function test_get_items_capabilities() {
```

Comment:
```suggestion
	/**
	 * @ticket 16841
	 */
	public function test_get_items_capabilities() {
```

:1234: Applies to each test